docs/troubleshoot/ze/ze-known-limits.md

```diff
@@ -0,0 +1,26 @@
+# Known Zowe Explorer limits
```

Comment:
I think "Known Zowe Explorer limitations" is clearer.

Reply:
good point, that was awkward wording. updated.